src/tests/Blef.GameLogic.Tests/PokerHandsHierarchy/PokerHandHierarchyTest.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Blef.GameLogic.PokerHands;
+using Blef.GameLogic.Tests.TestData;
+using NUnit.Framework;
+
+namespace Blef.GameLogic.Tests
+{
+    public class PokerHandHierarchyTest
+    {
+        [TestCaseSource(typeof(TestPokerHandsGenerator), nameof(TestPokerHandsGenerator.GeneratePokerHandsMatrix))]
+        public static void should_be_able_to_tell_which_one_pokerhand_is_stronger_than_another(int i, int j,
```

Comment:
Test TwoPairHandTests that was showing an explicit bug cannot be moved to big generator testing everything. I would never guess that now I need to go to test results (6920 tests!) and search for one particular case. 

Explicit is explicit. If the algorithm of generating TestCases will change (and it already changed as you said) we are not sure if we are covering the bug regression test.